aries-site/src/pages/templates/forms.mdx

```diff
@@ -80,7 +79,6 @@ Complicated processes where the user might need guidance at various stages, or w
 
 **Format and fields**
 
-- Have more than two optional fields.
```

Comment:
Shouldn't this guidance still apply? Aren't we trying to limit the amount of optional fields included in forms?

Reply:
I agree. While we don't have solid guidance of when "optional" should be used, so we're sticking to  asterisks being mandatory and the rest without asterisks being "optional".

Reply:
Agree but that's not what this section is talking about. This section is saying "Don't have more than two optional fields." and has nothing to do with marking as required or optional. I feel like that guidance is valid to stay.

Reply:
put back

Reply:
> Agree but that's not what this section is talking about. This section is saying "Don't have more than two optional fields." and has nothing to do with marking as required or optional. I feel like that guidance is valid to stay.

Agreed. Might help to have a hint of why though.